src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatRole.cs

```diff
@@ -11,21 +11,21 @@
 namespace Microsoft.Extensions.AI;
 
 /// <summary>
-/// Describes the intended purpose of a message within a chat completion interaction.
+/// Describes the intended purpose of a message within a chat interaction.
```

Comment:
This is a very different way to phrase it than the XML doc on the `ChatResponseUpdate.Role` property itself. TBH I prefer the phrasing here, since practically speaking we care about the purpose, and the notion of "authorship" is dubious at best (we don't literally care who/what wrote the messages, and the metadata about author identity it isn't necessarily true - it's certainly not authenticated/signed or anything like that; we only care what we're going to do with the messages).